packages/eas-cli/src/project/publish.ts

```diff
@@ -271,6 +277,7 @@ export async function uploadAssetsAsync(assetsForUpdateInfoGroup: CollectedAsset
   >(assetsWithStorageKey, asset => asset.storageKey);
 
   let missingAssets = await filterOutAssetsThatAlreadyExistAsync(uniqueAssets);
+  const uniqueUploadedAssetCount = missingAssets.length;
   const { specifications } = await PublishMutation.getUploadURLsAsync(
     missingAssets.map(ma => ma.contentType)
   );
```

Comment:
In the future we'll probably want to upload the assets in smaller batches, a few at a time, and show interactive progress.

Reply:
I do have the interactive progress hooked up. happy to add in the smaller batches functionality as well but that should probably be its own PR

Reply:
Agreed